api/Response.json

```diff
@@ -264,10 +304,10 @@
           "spec_url": "https://fetch.spec.whatwg.org/#ref-for-dom-response-error①",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "43"
```

Comment:
This is suspect, a case of https://github.com/mdn/browser-compat-data/issues/7843?

Reply:
It does look strange, but we have custom tests for these APIs, so this is accurate.

Reply:
Aha, sorry I didn't check the tests.